components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/java/org/wso2/carbon/identity/api/server/identity/governance/v1/core/ServerIdentityGovernanceService.java

```diff
@@ -302,6 +307,55 @@ public void updateGovernanceConnectorProperty(String categoryId, String connecto
         }
     }
 
+    /**
+     *
+     * Update multiple governance connector properties.
+     *
+     * @param categoryId                 Governance connector category id.
+     * @param multipleConnectorsPatchReq Governance connectors and properties to update.
+     */
+    public void updateGovernanceConnectorProperties(String categoryId,
+                                                    MultipleConnectorsPatchReq multipleConnectorsPatchReq) {
+
+        try {
+            IdentityGovernanceService identityGovernanceService = GovernanceDataHolder.getIdentityGovernanceService();
+            String tenantDomain = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain();
+
+            // Check whether the category ID exists.
+            CategoryRes category = getGovernanceConnectorCategory(categoryId);
+            if (category == null) {
+                throw handleNotFoundError(categoryId, GovernanceConstants.ErrorMessage.ERROR_CODE_CATEGORY_NOT_FOUND);
+            }
+
+            Map<String, String> configurationDetails = new HashMap<>();
+
+            // Loop through each connector in the request and add the properties to be updated to a hashmap.
+            for (ConnectorReq connectorReq : multipleConnectorsPatchReq.getConnectors()) {
+                String connectorId = connectorReq.getId();
+
+                // Check whether the connector ID exists.
+                ConnectorRes connector = getGovernanceConnector(categoryId, connectorId);
```

Comment:
```suggestion
                ConnectorResponse connector = getGovernanceConnector(categoryId, connectorId);
```

Reply:
`ConnectorRes` is an already existing class.